pages/common/git-secret.md

```diff
@@ -0,0 +1,36 @@
+# git-secret
```

Comment:
```suggestion
# git secret
```

Reply:
The package and repo are called git-secret, that's why I've went for git-secret.

Reply:
But that's not our naming scheme. Look at `tldr git bug` for example. That is also an extension to git and its title doesn't have the dash, we don't put it in subcommand pages.

Reply:
Fair enough...